bindings/go/src/fdb/tuple/tuple_test.go

```diff
@@ -0,0 +1,119 @@
+package tuple
+
+import (
+	"bytes"
+	"encoding/gob"
+	"flag"
+	"math/rand"
+	"os"
+	"testing"
+)
+
+var update = flag.Bool("update", false, "update .golden files")
+
+func loadGolden(t *testing.T) (golden map[string][]byte) {
+	f, err := os.Open("testdata/tuples.golden")
```

Comment:
Does this break if the test is run from a directory other than the directory where the test file is located in the source tree?

Reply:
Tests in Go are always run on the directory they reside, so assets can be loaded trivially. This is lifted straight from the "golden files" testing pattern on the standard library:

https://golang.org/src/cmd/gofmt/gofmt_test.go#L134

Reply:
By "always run" I mean that `go test` enforces this, regardless of where you're running `go test` from. 😄

Reply:
Oh, interesting, I didn't know that.